roles/people-ops.md

```diff
@@ -1,10 +1,23 @@
-# People operations (people-ops) role
+# People operations role ([PeopleOps])
 
-## Checklist for trialist onboarding
+**Contents**
+  1. [Onboarding](#onboarding)
+      1. [Trialist](#trialist)
+      1. [Permanent](#permanent-position)
+  3. [Offboarding](#offboarding)
 
-A member of the [PeopleOps] team will do the following to start the [onboarding](onboarding.md) process:
+## Onboarding
 
-* Send the <a href="mailto:?
+> *Reference document: [Onboarding](onboarding.md)*
+
+
+### Trialist
+
+When a new team member is hired on trial basis, a member of the [PeopleOps] team should do the following to start the [onboarding](onboarding.md) process:
```

Comment:
`team should do the following`: should? We need to start the onboarding process. It is not an option.

Reply:
You have raised an interesting point about language dynamics. I didn't feel `will` was correct here and yes `should` in the strictest sense does imply an option but often used as a 'soft way' of say it must be done. 

I did find a useful article on the subject that could be part of the handbook guidelines: http://reqexperts.com/blog/2012/10/using-the-correct-terms-shall-will-should/

Based on that I would agree with the article that `shall` is a much better alternative.